.travis.yml

```diff
@@ -5,45 +5,43 @@ rvm:
 jobs:
   include:
     - stage: install
-    - stage: install_rainforest_cli
-      script: 
-      # Install the latest stable version of rainforest-cli.
-      # This will allow us to trigger a Rainforest run after each deploy.
-      - wget "https://bin.equinox.io/c/htRtQZagtfg/rainforest-cli-stable-linux-amd64.tgz"
-      - tar xvf rainforest-cli-stable-linux-amd64.tgz
     - stage: run_unit_tests
       script:
       # Run unit tests
       - bundle exec rake
     - stage: run_rainforest_tests
-    # Trigger Rainforest run and wait for results
-    - rainforest run all --token $RAINFOREST_TOKEN --fg
-    - stage: deploy_staging
-      script:
-      # Deploy to our staging environment on Heroku
-      - provider: heroku
-        api_key:
-          secure: gGZwXnv/+n5J1qRzcbm8hQZxg9eToNPKNm0Vjs2S05kVzq67sPZD0MlfzUyx1ejqZTS7FWxkTWspAs3QTgJC7HsTXGY0qIenK/m9KkUKMpEgd35hMBLJx516kQgO1GAFkJSYp/AvIVN6krqs0Z96U0eshKdmWdINO6PTa95QLds=
-        app: rainforest-ci-sample-staging
-        on:
-          repo: rainforestapp/ci-sample
-          branch: develop
-    - stage: deploy_prod
       script:
-      # Deploy to our master environment on Heroku
-      - provider: heroku
-        api_key:
-          secure: gGZwXnv/+n5J1qRzcbm8hQZxg9eToNPKNm0Vjs2S05kVzq67sPZD0MlfzUyx1ejqZTS7FWxkTWspAs3QTgJC7HsTXGY0qIenK/m9KkUKMpEgd35hMBLJx516kQgO1GAFkJSYp/AvIVN6krqs0Z96U0eshKdmWdINO6PTa95QLds=
-        app: rainforest-ci-sample-prd
-        on:
-          repo: rainforestapp/ci-sample
-          branch: master
+      # Install the latest stable version of rainforest-cli.
+      # This will allow us to trigger a Rainforest run after each deploy.
+      - wget "https://bin.equinox.io/c/htRtQZagtfg/rainforest-cli-stable-linux-amd64.tgz"
+      - tar xvf rainforest-cli-stable-linux-amd64.tgz
+      - sudo mv rainforest /usr/local/bin/
```

Comment:
You don't have to move it to `/usr/local/bin` - you can just extract it and run it in the current directory.

Reply:
It doesn't work because of some idiosyncracy in Travis. Unless I explicitly move it into that file, Travis thinks it can't find rainforest. 

I tried just `rainforest` as well as trying `./rainforest`. no dice. the only way that travis could manage to find it was by doing it this way. 

Travis is an absolute hassle to work with.